src/dpsnotificationservice/src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
-
+spring.application.rabbitmq.host: ${RABBITMQ_HOST:localhost}
+spring.application.rabbitmq.port: ${RABBITMQ_PORT:5672}
+spring.application.rabbitmq.username: ${RABBITMQ_USERNAME:guest}
```

Comment:
do we want these things in the open?

Reply:
yes this is out of the box config for rabbit mq